runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneOverlay.java

```diff
@@ -106,7 +108,20 @@ public Dimension render(Graphics2D graphics)
 			.left("Points: ")
 			.right(StackFormatter.formatNumber(client.getVar(Varbits.NMZ_POINTS)))
 			.build());
-
+		if (config.totalPoints())
+		{
+			panelComponent.getChildren().add(LineComponent.builder()
+				.left("Total: ")
+				.right(StackFormatter.formatNumber(client.getVar(VarPlayer.NMZ_REWARD_POINTS) + client.getVar(Varbits.NMZ_POINTS)))
+				.build());
+		}
+		if (config.pointsPerHour())
```

Comment:
Flip the order?
Points
Points/hr
Total

That's just me, others might have different opinions.
Rationale is, first 2 are related to current session, 3rd is related to your account in general.